DigitalLearningSolutions.Data.Tests/Services/PasswordResetServiceTests.cs

```diff
@@ -187,31 +187,52 @@ public async Task Removes_reset_password_for_exact_users_matching_email()
         }
 
         [Test]
-        public void Trying_get_null_user_when_generating_welcome_email_should_throw_an_exception()
+        public void GenerateAndSendDelegateWelcomeEmail_with_null_user_should_throw_an_exception()
         {
             // Given
             A.CallTo(() => userService.GetUsersByEmailAddress(A<string>._)).Returns((null, new List<DelegateUser>()));
 
             // Then
             Assert.Throws<UserAccountNotFoundException>(
-                () => passwordResetService.GenerateAndSendDelegateWelcomeEmail("recipient@example.com", "example.com")
+                () => passwordResetService.GenerateAndSendDelegateWelcomeEmail("recipient@example.com", "A1", "example.com")
             );
         }
 
         [Test]
-        public void Trying_to_send_password_reset_when_generating_welcome_email_sends_email()
+        public void GenerateAndSendDelegateWelcomeEmail_with_incorrect_candidate_number_should_throw_an_exception()

```

Comment:
Can we specify what the user's candidate number is like in the corresponding `GenerateAndSendDelegateWelcomeEmail` test?

Reply:
I'm not sure what you mean by this? Make the "IncorrectCandidateNumber" string a const?